dune-project

```diff
@@ -1,2 +1,5 @@
-(lang dune 1.0)
+(lang dune 2.0)
+
+(allow_approximate_merlin)
```

Comment:
This is odd; the ppx_cstruct should be the only thing in that directory. Can we get rid of it, or did you find you needed it to get the build to compile without warnings?

Reply:
yep, I get the following warning without that feature in ```dune-project```,
```
File "ppx_test/errors/dune", line 6, characters 3-60:
6 |    (run %{bin:cppo} -V OCAML:%{ocaml_version} %{input-file})))
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Warning: .merlin generated is inaccurate. Cannot mix preprocessed and non
preprocessed specificiations.
Split the stanzas into different directories or silence this warning by
adding (allow_approximate_merlin) to your dune-project.
```

Reply:
Can fix that in a future revision if necessary. Thanks for checking!